### ISSUE TITLE
Process: integration into google/brotli

### ISSUE THREAD

**Issue by eustas** [andybalholm/brotli#23]
Hello.

 I wonder, if this work could be integrated into google/brotli.

As I understand, this repo is generated by c2go project. But:
 * it is unclear, if it required ad-hoc configuration (mentioned in c2go readme)
 * also it generated code license might be a tricky question; I'll investigate soon

But the first things go first - I'm asking for authors blessing to do that.

**Comment by silverwind** [comment]
It would be awesome if this module or some variant of it would find its way into the golang stdlib, e.g. `compress/brotli`.

**Comment by jnoordsij** [comment]
Am I correct in saying this is now available at the `google/brotli` upstream at https://github.com/google/brotli/blob/master/go/brotli/go.mod? And is there anyone with a clear view on particular differences between the two implementations?

**Comment by andybalholm** [comment]
I wasn't aware of that. It looks like it's a pure-Go Brotli decoder (but not encoder). It was authored by @eustas , the same one who opened this issue. I don't think it's directly derived from my code; as near as I can tell, it's a translation of the Java Brotli decoder (which is also translated to Kotlin, TypeScript, and JavaScript).

**Comment by jnoordsij** [comment]
Yeah you're right; I only took a brief look at first, but it only seems to do encoding indeed. Wondering if it will ever receive a continuation. Anyways it might be interesting for some people only interested in the decode part.
